File: 168.ExcelSheetColumnTitle.py
class Solution(object):
    def convertToTitle(self, n):
        """
        :type n: int
        :rtype: str
        """
        stack = []
        while n:
            number = (n - 1) % 26
            stack.append(chr(number + 65))
            n = (n - 1) / 26
        result = ""
        while stack:
            result = result + stack[-1]
            del stack[-1]
        return result

File: 168.ExcelSheetColumnTitle.cpp
class Solution {
public:
    string convertToTitle(int n) {
        string res;
        while (n > 0) {
            res += (n - 1) % 26 + 'A';
            n = (n - 1) / 26;
        }
        return string(res.rbegin(), res.rend());
    }
};